keras_nlp/models/t5/t5_presets.py

```diff
@@ -36,6 +36,7 @@
             "activation": "relu",
             "use_gated_activation": False,
             "layer_norm_epsilon": 1e-06,
+            "tie_embedding_weights": True,
```

Comment:
let's update the default for T5Backbone to `True`

Reply:
Changed the default, thanks!